NO TICKET: refactor mapped companies --> organizations

jacob-a-brown opened the pull request:
<!--
- Title the PR with the JIRA project & ticket number and short description (e.g., BDMS-123: Example display bug), or NO TICKET
- Keep sections short and scannable.
-->
### **Why**
_This PR addresses the following problem / context:_
- Two tables stored the same information that mapped **OwnersData.Company** to organizations, essentially storing the same type of data in two locations
- There are some duplicate organizations in the lexicon table
- `Santa Fe County` organizations should use a standardized naming convention

### **How**
_Implementation summary - the following was changed / added / removed:_
- Concatenated the two company --> organization mapping files to have one source of truth. By putting all mapped companies --> organizations into one file we can identify duplicates if and when they arise and use a standard name
- Updated the contact transfer to only use the single company --> organization mapping file
- Removed duplicate organizations from the lexicon file
- All `Santa Fe County` organizations now use the nomenclature `SFC`, such as `SFC`, `SFC, Santa Fe County Animal Shelter`, `SFC , Fire Facilities` etc


### **Notes**
_Any special considerations, workarounds, or follow-up work to note?_
- Use bullet points here

jirhiker committed:
Merge pull request #262 from DataIntegrationGroup/transfer

Transfer refactor for future reusability improvements

jacob-a-brown committed:
fix: remvove duplicate 'Sangre de Cristo Center' entry in lexicon

jacob-a-brown committed:
fix: remove duplicate 'Lamy MDWCA' entry from lexicon

jacob-a-brown committed:
fix: remove duplicate organizations from lexicon

jacob-a-brown committed:
refactor: put all mapped organizations into one file

keeping the organization mapping in a single file is easier for maintenance
and to standardize names in Ocotillo since it'll enable quick comparisons

jacob-a-brown committed:
fix: remove duplicate organizations from lexicon

jacob-a-brown committed:
refactor: use single table to map companies to organizations

this uses the new single table rather than two tables that contain
the same information

jacob-a-brown committed:
refactor: standardize Santa Fe County names in lexicon

use SFC for all santa fe county organizations for standardized nomenclature

jacob-a-brown committed:
note: write note about company --> organization mapping

jacob-a-brown committed:
fix: validate organization enum in contact transfer

this was originally used and I removed it in the refactor. I added it
back to ensure validation errors are caught if an organization is not
in the lexicon

ksmuczynski commented on transfers/data/owners_organization_mapper.json:
```diff
@@ -2,123 +2,187 @@
+	"Cebolleta Land Grant": "Cebolleta Land Grant",
 	"Cemex Plant": "Cemex, Inc",
 	"Cerro Community Center": "Cerro Community Center",
+	"Chama West Water Users Assn.": "Chama West Water Users Association",
```
Some values appear to abbreviate "Water User Association" to "WUA". This one is spelled out. If attempting to standardize, consider choosing one or the other.

jacob-a-brown replied:
I'll change to `WUA` to correspond with other implementations

ksmuczynski commented on transfers/data/owners_organization_mapper.json:
```diff
@@ -2,123 +2,187 @@
 	"Cemex Plant": "Cemex, Inc",
 	"Cerro Community Center": "Cerro Community Center",
+	"Chama West Water Users Assn.": "Chama West Water Users Association",
+	"Chamita Water Users Association": "Chamita MDWCA",
```
Is this the correct value? The key says "Water Users Association" but the value us "MDWCA".

jacob-a-brown replied:
When researching these organizations I couldn't find any WUAs, only the MDWCA
- https://nmrwa.org/members/chamita-mdwca/
- https://www.osa.nm.gov/wp-content/uploads/2019/12/additional-mutual-domestic.pdf

jacob-a-brown replied:
I'll update the definition in the lexicon

ksmuczynski reviewed (approved):
Nice work! Left a couple of very minor comments.

jacob-a-brown committed:
fix: update and standardize categories

Use consistent naming conventions for organizations in lexicon and data mapper.

jirhiker committed:
Merge branch 'transfer' into jab-contact-organization-refactor

jirhiker committed:
Formatting changes
